x-pack/legacy/plugins/index_management/public/app/services/documentation.ts

```diff
@@ -38,7 +37,7 @@ class DocumentationService {
   }
 
   public getTypeDocLink = (type: DataType | ConfigType, uri = 'main'): string | undefined => {
```

Comment:
I think this can be cleaned up more if it is no longer going to be used for the config doc links.

```
  public getTypeDocLink = (type: DataType, uri = 'main'): string | undefined => {
    const typeDefinition = TYPE_DEFINITION[type];

    if (!typeDefinition || !typeDefinition.documentation || !typeDefinition.documentation[uri]) {
      return undefined;
    }
    return `${this.esDocsBase}${typeDefinition.documentation[uri]}`;
  };
```